ui/src/overriden/components/Menu/RightMenuItem.js

```diff
@@ -0,0 +1,57 @@
+import { FrontSiteRoutes } from '@inveniosoftware/react-invenio-app-ils';
+import Qs from 'qs';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Dropdown, Menu, Responsive } from 'semantic-ui-react';
+import { config } from '../../../config';
+
+const onClickBookRequestLink = () => {
+  const params = Qs.parse(window.location.search);
+  const queryString = params['?q'];
+  return {
+    pathname: FrontSiteRoutes.documentRequestForm,
+    state: { queryString },
+  };
+};
+
+const dropdownEntries = (
+  <Dropdown.Menu>
+    <Dropdown.Item as="a" href="https://library.web.cern.ch/resources/remote">
+      External Online <br /> Resources
```

Comment:
I'm not sure why did you have to add `<br />` in the link name?

Reply:
Because otherwise the menu was very wide and the other links had too much white space on the right.